drivers/audio/dmic_mcux.c

```diff
@@ -18,6 +18,7 @@
 
```

Comment:
Please do the code formatting in a separate patch. A patch should do one single logical change.